public/main.js

```diff
@@ -614,12 +609,20 @@ const mapComp = Vue.component('map-comp', {
             } else if (this.mods.list.includes("noitavania")) {
                 mapName = Number(this.mods.ngp) > 0 ? "noitavania-new-game-plus" : "noitavania"
             }
-            let map = mapData[mapName]
+
+            // get map URL and topleft offset
+            const mapValues = this.mapData[mapName]
+            const map = {
+                urls: mapValues.map((x) => `${x.url.slice(0, -4)}_files/`),
```

Comment:
You could use something like `${x.url.replace(/\.dzi$/, '')}` to make the intent clearer here. In real code I'd be looking for validation of the (unknown/untrusted) input data, but the value of adding that here is minimal.